docs/ros/ros/api/overview.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: ROS 2 Robot API
+sidebar_label: Overview
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+
+The ROS 2 API is a standard naming scheme for ROS 2 topics and services. Every supported Clearpath
+robot uses this API, making it easy to use the robots interchangeably.
+
+
+## QoS Profiles {#qos-profiles}
+
+Topics and services in ROS 2 use Quality of Service (QoS) profiles to change communication policies. The QoS profile
+of a topic or service has several policies such as history, depth, reliability, durability, and more. For more details on 
+QoS settings, visit the [ROS 2 documentation](https://docs.ros.org/en/humble/Concepts/About-Quality-of-Service-Settings.html).
+
+For Clearpath platforms, the main policies that may change between different topics and services are reliability and durability. For reliability
+there are two options: *Best Effort*, and *Reliable*. A *Best Effort* reliability suggests that an attempt will be made to publish
+the data, but if the network is not robust then the data may be lost. On the other hand, *Reliable* guarantees that the data will be received.
+This may require that the data be sent multiple times.
+
+The durability of a QoS profile can be either *Transient Local* or *Volatile*. *Transient Local* means that data for published messages will
+be stored by the publisher even after it has been published. A new subscriber will receive the data even if a lot of time has passed since it was originally published.
+This durability is typically used for data that only needs to be published once, such as the robot description. The other durability policy is *Volatile* which does not
+stored old messages. The messages are published and only active subscribers will receive the data. New subscribers will have to wait for the next message to be published.
+Most topics will use a *Volatile* durability policy.
+
+It is important to check for [QoS compatibility](https://docs.ros.org/en/humble/Concepts/About-Quality-of-Service-Settings.html#qos-compatibilities) when interacting with ROS 2.
```

Comment:
https://docs.ros.org/en/humble/Concepts/Intermediate/About-Quality-of-Service-Settings.html#qos-compatibilities